schema/claim-result.schema.json

```diff
@@ -32,6 +32,9 @@
       "description": "The date created, as an ISO-8601 Extended Format date string, as specified in the ECMAScript standard",
       "type": "string"
     },
+    "custom": {
```

Comment:
Yikes! Thank you for adding this in, as I will be relying on it later for doing audits. 🙇‍♀️